eap/eap71-tx-recovery-s2i.json

```diff
@@ -101,6 +101,13 @@
             "value": "openshift",
             "required": true
         },
+        {
+            "displayName": "Project Namespace",
+            "description": "Project namespace where the template pods will be installed to. The value is used to define service account and role to permit listing pods from OpenShift API.",
+            "name": "NAMESPACE",
```

Comment:
We should try to remove this.  Having to specify the namespace directly is a bit of a usability problem, since it should default to the namespace within which the resources are being created.